src/conditioning.h

```diff
@@ -103,17 +103,12 @@ class Conditioning
   virtual void Tock();
 
  protected:
-  ///   @brief adds a material into the incoming commodity inventory
-  ///   @param mat the material to add to the incoming inventory.
-  ///   @throws if there is trouble with pushing to the inventory buffer.
-  void AddMat_(cyclus::Material::Ptr mat);
-
   /// @brief Move all unprocessed inventory to processing
   void BeginProcessing_();
 
 /// @brief move ready resources from processing to packaged after repackaging
   /// @param *** ADD HERE ***
-  void PackageMatl_();
+  void PackageMatl_(int size_package,std::map<std::string, std::map<std::string, double>> package_properties);
```

Comment:
is this line over the max line length?

Reply:
also, spaces after commas in the param list... 
```suggestion
  void PackageMatl_(int size_package, std::map<std::string, std::map<std::string, double>> package_properties);
```